uint32 marker_id
float32 x
float32 y
float32 z
float32 yaw
bool masked
---
bool success